OracleGoldenGate/README.md

```diff
@@ -1,8 +1,13 @@
-Oracle GoldenGate on Docker
-===============
+# Oracle GoldenGate on Docker
+
 Sample Docker build files to provide an installation of Oracle GoldenGate for DevOps users.
 
 ## Oracle GoldenGate 19c and earlier
 
 Provides Docker build files to create a Docker image for Oracle GoldenGate versions 12c through 19c.
 For more details, see [12c-19c/README.md](12c-19c/README.md).
+
+## Oracle GoldenGate 21c
+
+Provides Docker build files to create a Docker image for Oracle GoldenGate version 21c.
```

Comment:
Please replace Docker with container for the second instance, i.e. "Provides Docker build files to create a container image..."